rocAL/rocAL/source/meta_node_rotate.cpp

```diff
@@ -89,20 +90,20 @@ void RotateMetaNode::update_parameters(MetaDataBatch* input_meta_data)
             min_y = std::min(y1, std::min(y2, std::min(y3, y4)));
             max_x = std::max(x1, std::max(x2, std::max(x3, x4)));
             max_y = std::max(y1, std::max(y2, std::max(y3, y4)));
-            box.x = (min_x > 0) ? min_x : 0;
-            box.y = (min_y > 0) ? min_y : 0;
-            box.w = max_x - box.x;
-            box.h = max_y - box.y;
+            box.l = (min_x > 0) ? min_x : 0;
+            box.t = (min_y > 0) ? min_y : 0;
+            box.r = max_x ;
```

Comment:
compute direct here instead of using another set of variables (max_x and max_y)

Reply:
Resolved